tests/iotools/test_nasa_power.py

```diff
@@ -0,0 +1,88 @@
+import pandas as pd
+import pytest
+import pvlib
+from requests.exceptions import HTTPError
+
+
+@pytest.fixture
+def data_index():
+    index = pd.date_range(start='2025-02-02 00:00+00:00',
+                          end='2025-02-02 23:00+00:00', freq='1h')
+    return index
+
+
+@pytest.fixture
+def ghi_series(data_index):
+    ghi = [
+        0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 50.25, 184.2, 281.55, 368.3, 406.48,
+        386.45, 316.05, 210.1, 109.05, 12.9, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0
+    ]
+    return pd.Series(data=ghi, index=data_index, name='ghi')
+
+
+def test_get_nasa_power(data_index, ghi_series):
```

Comment:
```suggestion

@pytest.mark.remote_data
@pytest.mark.flaky(reruns=RERUNS, reruns_delay=RERUNS_DELAY)
def test_get_nasa_power(data_index, ghi_series):
```

All test functions that access internet APIs should have these two decorators :)

And add `from tests.conftest import RERUNS, RERUNS_DELAY` at the top of the file.